common/metrics/defs.go

```diff
@@ -622,8 +622,11 @@ const (
 	InputhostPubConnection
 	// InputhostLatencyTimer represents time taken by an operation
 	InputhostLatencyTimer
-	// InputhostReceiveMessageLatency is the latency to recieve a message
-	InputhostReceiveMessageLatency
+	// InputhostWriteMessageLatency is the latency from receiving a message from stream to returning ack back to stream
+	InputhostWriteMessageLatency
+	// InputhostWriteMessageExcludeSocketLatency is the latency from receiving a message from stream to getting ack from replicas
+	// the only difference with InputhostWriteMessageLatency is this metrics excludes the latency for writing ack back to the socket
+	InputhostWriteMessageExcludeSocketLatency
```

Comment:
Suggest `InputhostWriteMessageBeforeAckLatency`. "Exclude socket" is weird because it does need to talk to store hosts over socket.